README.md

```diff
@@ -85,7 +85,7 @@
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [https://danielmatama.github.io/matama-s-portfolio/](https://yourdeployedapplicationlink.com)
```

Comment:
- [ ] Kindly format this properly as I am currently unable to access your site when I clicked it on your readme file. The proper way I would advise is `[Live Demo](https://danielmatama.github.io/matama-s-portfolio/)`. You can replace what you wrote with the one I highlighted and you will be fine.